### ISSUE TITLE
508-defect-2 [SCREENREADER]: Screenreader SHOULD announce the accurate number of items

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12060]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The screen reader **should** announce the correct number of items. As shown in the screenshot, although there are ten items, the screen reader announces there are seven.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Environment

* Operating System: Mac
* Browser: Safari
* Screenreading device: VoiceOver
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/education/yellow-ribbon-participating-schools/?contributionAmount=unlimited in browser
2. Start screenreading device listed in Environment
3. Navigate to the Search button, and select it
4. Verify that "heading level 2 7 items Displaying 1 through 10 of 2216 results, main" is read

## Screenshots or Trace Logs

![Screen Shot 2020-08-04 at 12 25 06 PM (2)](https://user-images.githubusercontent.com/57469/89348242-38788c80-d67a-11ea-8ace-f3478ff9811c.png)

**Comment by 1Copenut** [comment]
@jenstrickland I have a feeling Safari + VO is announcing 7 items as the child elements (elementNodes and textNodes) inside the heading. I took a quick screen shot and counted seven items, but I'm a little surprised the text is broken up into individual strings. I wonder if the React component could assemble the string first, then read it out as one item in the heading. I agree this could be confusing. LMK if you want to discuss this in the morning.

---
![Screen Shot 2020-08-17 at 5 22 42 PM](https://user-images.githubusercontent.com/934879/90450167-aa0afe80-e0ae-11ea-9dca-6951de3430bc.png)

**Comment by jenstrickland** [comment]
@1Copenut This sounds logical for where the issue may originate, and will be helpful for the devs as they work on this ticket. **Thank you!**

**Comment by MarciMcGuireGCIO** [comment]
Moving to a consolidated 508 epic for Yellow Ribbon to clean up our backlog.

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO I can use the same solution to what we had to do for Find Forms. 
The only thing is, after a bunch of research @joshkimux and I found out that there is a known bug with Fire Fox. But the other browsers will be fixed.

**Comment by zacharymorel** [comment]
@joshkimux Can you give this a look over now that it is live on 
https://www.staging.va.gov/education/yellow-ribbon-participating-schools/

**Comment by joshkimux** [comment]
<img width="987" alt="Screen Shot 2021-06-25 at 2 10 42 PM" src="https://user-images.githubusercontent.com/14154792/123467965-23248d80-d5bf-11eb-85b7-99bec6c6a7b1.png">
Looking good 🥳

**Comment by MarciMcGuireGCIO** [comment]
Awesome!  Thanks, @zacharymorel and @joshkimux !
